classes/submission/SubmissionFile.inc.php

```diff
@@ -66,6 +94,22 @@ function setFileId($fileId) {
 	}
 
 	/**
+	 * Get the locale of the submission.
+	 * @return string
```

Comment:
I would suggest noting in the getter/setters that this is not properly a property of the submission file (e.g. so devs know it won't be persisted to the DB with the usual update function).